src/check.rs

```diff
@@ -27,20 +27,27 @@ pub fn check(chip: Chip) {
         },
     );
 
+    let rust_toolchain = if chip.is_xtensa() { "esp" } else { "stable" };
+
     let espflash_version = get_version("espflash", &[]);
     let probers_version = get_version("probe-rs", &[]);
 
     println!("\nChecking installed versions");
-    print_result("Rust", check_version(rust_version, 1, 84, 0));
+    print_result(
+        &format!("Rust ({rust_toolchain})"),
+        check_version(rust_version, 1, 84, 0),
+    );
     print_result("espflash", check_version(espflash_version, 3, 3, 0));
     print_result("probe-rs", check_version(probers_version, 0, 25, 0));
 }
 
 fn print_result(name: &str, check_result: CheckResult) {
     match check_result {
         CheckResult::Ok => println!("🆗 {}", name),
-        CheckResult::WrongVersion => println!("🛑 {}", name),
-        CheckResult::NotFound => println!("❌ {}", name),
+        CheckResult::WrongVersion => {
+            println!("🛑 {} (version requirements are not satisfied)", name)
+        }
+        CheckResult::NotFound => println!("❌ {} (not found)", name),
```

Comment:
At least let's print the minimum required version, also how to install/update. Also, a missing probe-rs may or may not be a hard error depending on the project requirements, and the message printed should reflect that.

Reply:
Also somewhat relates to #171

but yeah - a hint if probe-rs is not really needed might help